src/TArea.h

```diff
@@ -82,6 +84,8 @@ class TArea
     TRoomDB* mpRoomDB;
     bool mIsDirty;
     QMap<QString, QString> mUserData;
+    // This has been moved from the TMap class:
```

Comment:
Don't need to have the historical information on refactoring in Git, the theme still stays